examples/notebooks/tour-of-beam/dataframes.ipynb

```diff
@@ -0,0 +1,744 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 2,
+  "metadata": {
+    "colab": {
+      "name": "Beam DataFrames",
+      "provenance": [],
+      "collapsed_sections": [],
+      "toc_visible": true
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "source": [
+        "#@title ###### Licensed to the Apache Software Foundation (ASF), Version 2.0 (the \"License\")\n",
```

Comment:
```suggestion
        "###### Licensed to the Apache Software Foundation (ASF), Version 2.0 (the \"License\")\n",
```

nit: This shows up as a section in the table of contents, which looks a little odd. I think this fixes it?

Reply:
Yeah, it shows up in the table of contents, but by adding `@title` we can collapse the cell to only show up the summary. It's a little less intrusive while reading the notebook.